<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="BitmapWidget" name="bitmap" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>171</width>
     <height>161</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRun">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonStopReset">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop/Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonStep">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRegisters">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Registers</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="R_1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>R1</string>
   </property>
  </widget>
  <widget class="QLabel" name="R_0">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>R0</string>
   </property>
  </widget>
  <widget class="QLabel" name="R_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>R3</string>
   </property>
  </widget>
  <widget class="QLabel" name="R_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>R2</string>
   </property>
  </widget>
  <widget class="QLabel" name="R_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>R4</string>
   </property>
  </widget>
  <widget class="QLabel" name="R_5">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>R5</string>
   </property>
  </widget>
  <widget class="QLabel" name="R_6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>100</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>R6</string>
   </property>
  </widget>
  <widget class="QLabel" name="R_7">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>120</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>R7</string>
   </property>
  </widget>
  <widget class="QLabel" name="val_R_0">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0x00</string>
   </property>
  </widget>
  <widget class="QLabel" name="val_R_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0x00</string>
   </property>
  </widget>
  <widget class="QLabel" name="val_R_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0x00</string>
   </property>
  </widget>
  <widget class="QLabel" name="val_R_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>60</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0x00</string>
   </property>
  </widget>
  <widget class="QLabel" name="val_R_5">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0x00</string>
   </property>
  </widget>
  <widget class="QLabel" name="val_R_6">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0x00</string>
   </property>
  </widget>
  <widget class="QLabel" name="val_R_7">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0x00</string>
   </property>
  </widget>
  <widget class="QLabel" name="val_R_1">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0x00</string>
   </property>
  </widget>
  <widget class="QLabel" name="flags">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Flags</string>
   </property>
  </widget>
  <widget class="QLabel" name="val_flags">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>b00000000</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableCommand">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>271</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="TextEditAddress">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonGo">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>210</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
  </widget>
  <widget class="QLabel" name="EnterAddress">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter address</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BitmapWidget</class>
   <extends>QWidget</extends>
   <header>bitmapwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
